ansible-test/dockerd.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash 
+#/usr/local/bin/dockerd.sh 
```

Comment:
Did you actually copy the existing file from the `docker` package?
If so, you should write a comment at the beginning of this file that this file is copied from that file.

Also, the shebang, i.e., the first line `#!/bin/bash` should be replaced with just `#!/bin/sh`. I can explain the rationale when we sync today.